packages/css/src/types.ts

```diff
@@ -50,7 +50,7 @@ interface CSSOthersObjectForCSSObject {
 /**
  * Map all nested selectors
  */
-interface CSSSelectorObject {
+export interface CSSSelectorObject {
```

Comment:
Are you using it in this PR?
It's a part of StyleSystemObject, so it makes sense to export it.
I'm just asking if I missed something.

Reply:
Good point and I was using `SystemStyleObject` at some point but then I refactored it and while doing that I've noticed that we weren't exporting all the possible types of:
https://github.com/system-ui/theme-ui/blob/076cd7b5cab2a8a58f96da9f148a33590588f169/packages/css/src/types.ts#L467-L472
And this would throw a typecheck error saying that some types couldn't be found (since they weren't exported).

Are you ok having this here or shall I raise a different PR with it @hasparus ?

Reply:
I'm working on this part in https://github.com/system-ui/theme-ui/pull/910, because our type errors on `sx` are wonky.
Quite possibly, these types will change to something closer to @emotion/core Interpolation types.
Just a disclaimer in case you import them in other projects.